### PR TITLE
Chapter 4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by Thuyha96**
ready

### CODE REVIEW

**Comment by Thuyha96** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title(page_title = '')
```
Comment (thanhhungchu95) [inline app/helpers/application_helper.rb]:
```suggestion
  def full_title page_title = ""
```

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (thanhhungchu95) [inline app/helpers/application_helper.rb]: i18n

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+  <title><%= full_title(yield(:title)) %></title>
```
Comment (thanhhungchu95) [inline app/views/layouts/application.html.erb]:
```suggestion
  <title><%= full_title yield :title %></title>
```

**Comment by Thuyha96** [comment]
ready

**Comment by Thuyha96** [comment]
ready

**Review by thanhhungchu95 — COMMENTED** [review]
(merged)

**Pull request merged**